packages/features/insights/context/FiltersProvider.tsx

```diff
@@ -69,13 +69,20 @@ export function FiltersProvider({ children }: { children: React.ReactNode }) {
   const [selectedFilter, setSelectedFilter] = useState<FilterContextType["filter"]["selectedFilter"]>(
     filterParsed ? [filterParsed] : null
   );
+  const [isOrg, setIsOrg] = useState<FilterContextType["filter"]["isOrg"]>(false);
+
   const [selectedTeamName, setSelectedTeamName] =
     useState<FilterContextType["filter"]["selectedTeamName"]>(null);
   const [dateRange, setDateRange] = useState<FilterContextType["filter"]["dateRange"]>([
     startTimeParsed ? dayjs(startTimeParsed) : dayjs().subtract(1, "month"),
     endTimeParsed ? dayjs(endTimeParsed) : dayjs(),
     "t",
   ]);
+  const [initialConfig, setInitialConfig] = useState<FilterContextType["filter"]["initialConfig"]>({
+    userId: null,
+    teamId: null,
+    isOrg: false,
+  });
```

Comment:
Added this so I can save initialConfig and reinstate it when using clear filters button.